mnemonic/mnemonic.py

```diff
@@ -250,15 +250,18 @@ def to_seed(cls, mnemonic, passphrase=""):
         return stretched[:64]
 
     @classmethod
-    def to_hd_master_key(cls, seed):
+    def to_hd_master_key(cls, seed, network='main'):
         if len(seed) != 64:
             raise ValueError("Provided seed should have length of 64")
 
         # Compute HMAC-SHA512 of seed
         seed = hmac.new(b"Bitcoin seed", seed, digestmod=hashlib.sha512).digest()
 
         # Serialization format can be found at: https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#Serialization_format
-        xprv = b"\x04\x88\xad\xe4"  # Version for private mainnet
+        if network == 'main':
```

Comment:
if testnet ... elif: ...

Reply:
@prusnak Thank you for your review.
I modified.